src/vstart.sh

```diff
@@ -159,6 +159,7 @@ smallmds=0
 short=0
 crimson=0
 ec=0
+cephexporter=0
```

Comment:
Shouldn't we enable this by default, in order to maximize test coverage?

Reply:
I doubt it -- vstart clusters should be as minimal as possible.  Folks doing development on e.g. OSD wouldn't bother with a Prometheus instance that makes exporter useful.  That said, if there is a strong desire to enable exporter by default, more changes would be required: at least a) automatic disabling of exporter if it's not built and b) an option to disable exporter even if it's built.

Reply:
Make sense! Thanks @idryomov !